README.md

```diff
@@ -72,9 +82,29 @@ By default, the stack exposes the following ports:
 * 9300: Elasticsearch TCP transport
 * 5601: Kibana
 
-*WARNING*: If you're using *boot2docker*, you must access it via the *boot2docker* IP address instead of *localhost*.
+*WARNING*: If you're using `boot2docker`, you must access it via the `boot2docker` IP address instead of `localhost`.
+
+*WARNING*: If you're using *Docker Toolbox*, you must access it via the `docker-machine` IP address instead of `localhost`.
+
+# Create Indexes
```

Comment:
You might want to merge that with the line 77 (_"Refer to 'Connect Kibana with Elasticsearch' for detailed instructions about the index pattern configuration."_)

The wording is not 100% accurate, you're creating an index **pattern** here.